src/wp-includes/link-template.php

```diff
@@ -1597,16 +1597,23 @@ function get_delete_post_link( $post = 0, $deprecated = '', $force_delete = fals
  * @since 2.3.0
  *
  * @param int|WP_Comment $comment_id Optional. Comment ID or WP_Comment object.
+ * @param string         $context    Optional. Could be output the '&' character or 'display', Default '&amp;'.
```

Comment:
This isn't really correct as a description for this parameter. The default for the parameter is not `&amp;`, that's just the output that the parameter controls. For what it's worth, even the description of the parameter on `get_edit_post_link()` is a bit confusing.

I think this would be more clear:
```suggestion
 * @param string         $context    Optional. Context in which the URL should be used. Either 'display', 
 *                                   to include HTML entities, or 'display'. Default 'display'.
```